lib/commands/element.js

```diff
@@ -9,6 +9,10 @@ function toBool (s) {
   return _.isString(s) ? (s.toLowerCase() === 'true') : !!s;
 }
 
+commands.active = async function () {
```

Comment:
please add a name to the anonymous function - it makes easier to debug the the stuff:

`async function ()` -> `async function active ()`

Reply:
Right, fixed. Sorry for trouble with w3c stuff.

Reply:
np, that is why we have the review process :)